utils/python/CIME/SystemTests/rep.py

```diff
@@ -0,0 +1,22 @@
+"""
+Implementation of the CIME REP test
+
+This test verifies that two identical runs give bit-for-bit results
+"""
+
+from CIME.SystemTests.system_tests_compare_two import SystemTestsCompareTwo
+
+class REP(SystemTestsCompareTwo):
```

Comment:
Is REP a bad name? I realized that it's easy to mistype ERP as REP and vice versa. Is it time to start allowing longer test names? ("REPRODUCE"???)